test/beetle/base_test.rb

```diff
@@ -26,7 +26,7 @@ def setup
     end
 
     test "server should be initialized" do
-      assert_equal @bs.servers.first, @bs.server
+      assert @bs.server
```

Comment:
this spec failed when having multiple servers configured as they one of them is chosen randomly. The spec should only assert there is *a* server but not which one.

Reply:
the tests pass so ...